sdgym/summary.py

```diff
@@ -190,12 +201,41 @@ def add_sheet(dfs, name, writer, cell_fmt, index_fmt, header_fmt):
         worksheet.set_column(idx, idx, width + 1, fmt)
 
 
+def _find_category(synthesizer, categories):
+    for category, category_synthesizers in categories.items():
+        for category_synthesizer in category_synthesizers:
+            if category_synthesizer in synthesizer.lower():
+                return category
+
+    return None
+
+
+def _add_summary_categories(summary_data):
+    categories = {
```

Comment:
I think this should be made a module level constant, and then we do not need to pass it as an argument.